contributors/guide/github-workflow.md

```diff
@@ -74,6 +74,8 @@ git checkout -b myfeature
 Then edit code on the `myfeature` branch.
 
 #### Build
+This workflow is process-specific; for detailed build instructions for [kubernetes/kubernetes](https://github.com/kubernetes/kubernetes) please [see here](https://github.com/kubernetes/community/blob/master/contributors/devel/development.md#building-kubernetes-on-a-local-osshell-environment).
```

Comment:
Use "quick start" instead of "Detailed" since that is literally how the next section starts